### PR TITLE
fix: padding-right for right-pane-tabs 

### PR DESCRIPTION
**Pull request by trsvchn**
### Description

Fix # 

Fixes padding on the right for the tab panel.

### Additional context

Screenshot from 2021-05-26 08-53-51
Screenshot from 2021-05-26 08-53-55

---

### What is the purpose of this pull request?

- [ ] Bug fix
- [ ] New feature
- [x] Other

### CODE REVIEW

**Comment by ydcjeff** [comment]
We could also do with `margin-right`. Then we don't need to wrap in another `div`

**Comment by trsvchn** [comment]
@ydcjeff could you please show how to do it? Maybe via suggestion

File: src/components/PaneRight.vue [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-      <div
-        v-for="tab in tabs()"
-        :key="tab"
-        class="right-pane-tab"
-        :class="{ active: currentTab === tab }"
-        @click="currentTab = tab"
-      >
-        <span class="iconify" :data-icon="getFileType(tab)"></span>
-        {{ tab }}
+    <div class="right-pane-tabs-container">
```
Comment (ydcjeff) [inline src/components/PaneRight.vue:3]:
```suggestion
```

File: src/components/PaneRight.vue [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+        <div
+          v-for="tab in tabs()"
+          :key="tab"
+          class="right-pane-tab"
+          :class="{ active: currentTab === tab }"
+          @click="currentTab = tab"
+        >
+          <span class="iconify" :data-icon="getFileType(tab)"></span>
+          {{ tab }}
+        </div>
```
Comment (ydcjeff) [inline src/components/PaneRight.vue:14]:
```suggestion
```

File: src/components/PaneRight.vue [hunk 3]
```diff
@@ -77,7 +79,7 @@ export default {
 .right-pane-tabs::-webkit-scrollbar {
   display: none;
 }
-.right-pane-tabs,
+.right-pane-tabs-container,
```
Comment (ydcjeff) [inline src/components/PaneRight.vue:82]:
```suggestion
.right-pane-tabs,
```

File: src/components/PaneRight.vue [hunk 4]
```diff
@@ -77,7 +79,7 @@ export default {
...
 .right-pane-contexts {
   padding-right: 1.5rem;
```
Comment (ydcjeff) [inline src/components/PaneRight.vue:84]:
```suggestion
   margin-right: 1.5rem;
```

Comment (trsvchn) [reply src/components/PaneRight.vue:84]: Well, it was quite simple!
Thanks @ydcjeff !

**Pull request merged**